### ISSUE TITLE
Allow to download the generated bitstream?

### ISSUE THREAD

**Issue by fallen** [webfpga/icestorm-server#1]
Would that be desirable?

**Comment by ryanmjacobs** [comment]
I'm a little confused. The command-line applications already allow that.
Are you talking about the frontend website?

**Comment by fallen** [comment]
Yes, the web app allows to generate bitstream and to flash a device, it would be nice to just add a way to save/download the bitstream (I think it is already received in a JSON string anyway)

**Comment by fallen** [comment]
But I agree that it's just bonus since one can anyway just run the CLI locally

**Comment by fallen** [comment]
Don't hesitate to close if it's not relevant at all.

**Comment by ryanmjacobs** [comment]
Don’t worry, it is definitely relevant. We’re working on a full site revamp
that should be live soon. Stay tuned!
